tests/HJSONParserTest.php

```diff
@@ -10,7 +10,7 @@
 class HJSONParserTest extends TestCase
 {
 
-    protected function setUp(): void
+    public function setUp()
```

Comment:
According to the [PHPUnit fixture sample code](https://phpunit.readthedocs.io/en/8.0/fixtures.html?highlight=setUp), the `setUp` method should be `protected`, not `public`.

Reply:
Here is the [fixture sample code](https://phpunit.de/manual/6.5/en/fixtures.html) for `PHPUnit 6.5`.

It's also the `protected`, not `public`.